packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -266,9 +272,15 @@ Content-Type: text/json;charset=UTF-8
 
 ### Updating a specific user (PUT)
 
+![Flowchart - update user (PUT)](/img/oin/scim_flow-user-update-put.png "Simple flow diagram for updating a user with a PUT operation")
+
 Updating a user refers to modifying an attribute in the Okta user profile that is mapped with an attribute in the SCIM application.
 
-To update a user, the functionality needs to be enabled in Okta. In the Okta Admin Console, select the SCIM application from your list of applications. Under the **Provisioning** tab, click **To App**. In the **Update User Attributes** option, click **Enable** and then **Save**.
+To update a user, the functionality needs to be enabled in Okta. In the Okta Admin Console:
```

Comment:
To update a user, you need to enable the functionality in the Okta Admin Console: